lesson-01-servlet-api/src/main/java/ru/geekbrains/BootStrapListaner.java

```diff
@@ -0,0 +1,25 @@
+package ru.geekbrains;
+
+import ru.geekbrains.persist.User;
+import ru.geekbrains.persist.UserRepository;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
+@WebListener
+public class BootStrapListaner implements ServletContextListener {
+
+    @Override
+    public void contextInitialized(ServletContextEvent sce) {
+        ServletContext servletContext= sce.getServletContext();
+
+        UserRepository userRepository = new UserRepository();
+        servletContext.setAttribute("userRepository", userRepository);
+
+        userRepository.Insert(new User("Вася"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.